src/Screens/Products/index.js

```diff
@@ -0,0 +1,101 @@
+import React from 'react'
+import { useEffect, useState } from 'react';
+import { Link } from 'react-router-dom';
+import styles from './products.module.css';
+
+
+
+
+const Products = (props) => {
+  
+  const [products, setProducts] = useState([]);
+
+  useEffect(() => {
+    async function fetchProducts() {
+      try {
+        const response = await fetch(`${process.env.REACT_APP_API_URL}/products/`);
+        console.log(response);
+        const json = await response.json();
+        console.log(json);
+        setProducts(json);
+      } catch (error) {
+        alert('Could not GET Products.', error);
+      }}
+    fetchProducts();
+  }, []);
+
+  const deleteProduct = async (id) => {
```

Comment:
same logic of fetch products, you shouldn't execute async data handling inside the screen, move this to thunks and execute it here when the user wants to delete a product